src/views/poll.blade.php

```diff
@@ -0,0 +1,18 @@
+@extends('larapoll::layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="flex-center position-ref full-height">
+        @if ($errors->any())
+            <div class="alert alert-danger">
+                <ul>
+                    @foreach ($errors->all() as $error)
+                        <li>{{ $error }}</li>
+                    @endforeach
+                </ul>
+            </div>
+        @endif
+        {{ PollWriter::draw($poll->id) }}
```

Comment:
There is no need to call the draw method, there must be just a listing of options. with no need to vote.

Reply:
I don't understand what do you mean.
I used it from the `readme.md` file

Reply:
This method is to print the signle poll for voting not just to preview.

Reply:
Yes, I'm adding a new feature for showing a single poll.
I also added links on polls listing at the route `admin_polls/admin/polls`